src/card.css

```diff
@@ -0,0 +1,124 @@
+.page {
+  font-family: "Roboto", sans-serif;
+}
+
+.page__body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  width: 166px;
+
```

Comment:
You can use   box-sizing: border-box; and then use width: 200px. In this case, the width will be calculated taking into account the internal indents